src/layer/Popup.js

```diff
@@ -438,12 +436,12 @@ Layer.include({
 
 	// @method togglePopup(): this
 	// Opens or closes the popup bound to this layer depending on its current state.
-	togglePopup: function (target) {
+	togglePopup: function () {
 		if (this._popup) {
 			if (this._popup._map) {

```

Comment:
Here is the check if the popup has the map property `this._popup._map` and in `_openPopup` the check is `if (this._map.hasLayer(this._popup))`

Reply:
Yes, but that is necessary in both places